test/functional/Java19andUp/build.xml

```diff
@@ -50,7 +50,7 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 		<echo>===destdir:                      ${DEST}</echo>
 
 		<if>
-			<equals arg1="${JDK_VERSION}" arg2="19"/>
+			<matches string="${JDK_VERSION}" pattern="^(19|20)$$"/>
```

Comment:
This should be more general; we shouldn't need to revisit this for jdk21, jdk22, etc. See lines 84-86 for a better condition.

Reply:
The rationale behind this is that 19 and 20 have Loom as a preview but future versions may not. @tajila for clarification
Also the test won't be compiled for versions < 19

Reply:
That condition could be condensed to
```
	<not>
		<matches string="${JDK_VERSION}" pattern="^([0-9]|1[0-8])$$" />
	</not>
```

Reply:
The features that we are testing in 19 are preview for JDK19 and JDK20 (according to the current code base) but they will not remain preview indefinitely. Its possible that by jdk21 the preview flag will not be needed anymore.

Reply:
If we seek to control the use of the `--enable-preview` option, then it should be just that that is version sensitive; the `<javac>` element should be unconditional.

Reply:
> --enable-preview option, then it should be just that that is version sensitive

I seem to recall that we can't do conditionals inside the javac blocks

Reply:
That's right, but a `<condition>` element can be used to define the value of the `line` attribute of the `<compilerarg>` element.

Reply:
Updated